src/GeositeFramework/template_index.html

```diff
@@ -232,14 +164,14 @@
     </form>
 
     <script type="text/template" id="template-pane">
-        @if ({{ singlePluginMode }}) {
+        {% if singlePluginMode %}
         <text>
         <div id="single-plugin-mode-help-container">
             Help
         </div>
         </text>
```

Comment:
We should make an issue to remove these `<text>` tags which I think must be valid c# templating but aren't valid html. However it's out of scope for this issue -- just wanted to make sure it didn't get lost.